### PR TITLE
Add Package Manager Console Support

### PR DESCRIPTION
**Pull request by lajones**
Fixes issue #1674

### CODE REVIEW

File: src/EntityFramework.Commands/project.json [hunk 1]
```diff
@@ -7,25 +7,14 @@
...
-                "Microsoft.AspNet.Hosting": "1.0.0-*",
-                "Microsoft.Framework.CommandLineUtils": { "version": "1.0.0-*", "type": "build" }
-            }
-        },
-        "dnxcore50": {
-            "dependencies": {
-                "Microsoft.AspNet.Hosting": "1.0.0-*",
-                "Microsoft.Framework.CommandLineUtils": { "version": "1.0.0-*", "type": "build" }
-            }
-        }
```
Comment (bricelam) [inline src/EntityFramework.Commands/project.json]: Don't check these removals in :wink:

Comment (lajones) [inline src/EntityFramework.Commands/project.json]: Thought I'd reverted them all - missed that one. Will fix - thanks!

**Comment by bricelam** [comment]
:shipit:

**Comment by lajones** [comment]
Checked in - commit baed9b7.

**Pull request closed without merging**